### DISCUSSION TITLE
How to display all columns in a select statement

### DISCUSSION THREAD

**Discussion by meswaramoorthy** [duckdb/duckdb#6745] [Q&A] [answered]
Hi,

When the data has more columns than 10 or so and you perform a select, the output doesnot display all column insteam truncates the output with ...

Is there a setting or argument to enable the output to display all the columns?

**Comment by Alex-Monahan** [comment, marked as answer]
Howdy! If you are using the CLI, check out the `.mode` command!
https://duckdb.org/docs/api/cli

**Reply by meswaramoorthy** [reply]
It does help to see all the values using mode table. But it is not as pretty as duckbox.

**Reply by Tishj** [reply]
The `.maxwidth` option was created to support this even for duckbox :)
